dialogue-example/src/main/java/com/palantir/dialogue/example/SampleServiceClient.java

```diff
@@ -77,7 +77,7 @@ public HttpMethod httpMethod() {
     };
 
     /** Returns a new blocking {@link SampleService} implementation whose calls are executed on the given channel. */
-    public static SampleService blocking(Channel channel, ConjureRuntime runtime) {
+    public static SampleService blocking(Channel channel, ConjureRuntime runtime, Duration readTimeout) {
```

Comment:
wondering if it'd make more sense for this to be a builder-pattern thing or possibly for the third arg to be some kind of settings object

Reply:
I'll add a TODO, think that'd be a separate PR.